eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -87,11 +88,17 @@ def lsqfitter(lc, model, meta, calling_function='lsq', **kwargs):
 
     # Compute reduced chi-squared
     chi2red = computeRedChiSq(lc, model, meta, freenames)
-
+    
     print('\nLSQ RESULTS:')
+    # if lc.share:
     for freenames_i, fit_params_i in zip(freenames, fit_params):
         print('{0}: {1}'.format(freenames_i, fit_params_i))
-    print()
+    # else:
```

Comment:
If you're done with this commented code, you should remove it

Reply:
Done